dtfs-central-api/src/helpers/calculate-ukef-share-of-utilisation.test.ts

```diff
@@ -0,0 +1,14 @@
+import { calculateUkefShareOfUtilisation } from './calculate-ukef-share-of-utilisation';
+
+describe('calculateUkefShareOfUtilisation', () => {
+  it('should return the ukef share of the utilisation', () => {
```

Comment:
can we have a test that tests the rounding as well?

Reply:
done